delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/organizations/OrganizationDeleterSpec.scala

```diff
@@ -0,0 +1,115 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.organizations
+
+import cats.syntax.all._
+import ch.epfl.bluebrain.nexus.delta.kernel.utils.UUIDF
+import ch.epfl.bluebrain.nexus.delta.sdk.ConfigFixtures
+import ch.epfl.bluebrain.nexus.delta.sdk.generators.ProjectGen.defaultApiMappings
+import ch.epfl.bluebrain.nexus.delta.sdk.organizations.model.{Organization, OrganizationCommand}
+import ch.epfl.bluebrain.nexus.delta.sdk.organizations.model.OrganizationRejection.{OrganizationNonEmpty, OrganizationNotFound}
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.Projects.FetchOrganization
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.{ProjectsConfig, ProjectsFixture}
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.ProjectRejection.WrappedOrganizationRejection
+import ch.epfl.bluebrain.nexus.delta.sdk.projects.model.{ApiMappings, ProjectFields}
+import ch.epfl.bluebrain.nexus.delta.sourcing.{GlobalEventLog, MD5}
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{Identity, Label, ProjectRef}
+import ch.epfl.bluebrain.nexus.delta.sourcing.offset.Offset
+import ch.epfl.bluebrain.nexus.testkit.IOFixedClock
+import ch.epfl.bluebrain.nexus.testkit.bio.BioSuite
+import doobie.implicits._
+import doobie.util.update.Update0
+import monix.bio.{IO, Task, UIO}
+import munit.AnyFixture
+
+import java.util.UUID
+
+class OrganizationDeleterSpec extends BioSuite with IOFixedClock with ConfigFixtures {
+
+  private val org     = Label.unsafe("org")
+  private val orgUuid = UUID.randomUUID()
+
+  private def fetchOrg: FetchOrganization = {
+    case `org` => UIO.pure(Organization(org, orgUuid, None))
+    case other => IO.raiseError(WrappedOrganizationRejection(OrganizationNotFound(other)))
+  }
+
+  private val config         = ProjectsConfig(eventLogConfig, pagination, cacheConfig, deletionConfig)
+  private val projectFixture = ProjectsFixture.init(fetchOrg, defaultApiMappings, config)
+
+  override def munitFixtures: Seq[AnyFixture[_]] = List(projectFixture)
+
+  private lazy val (xas, projects) = projectFixture()
+  private lazy val orgDeleter      = OrganizationDeleter(xas)
+  private val projRef              = ProjectRef.unsafe("org", "myproj")
+  private val fields               = ProjectFields(None, ApiMappings.empty, None, None)
+  private lazy val orgs            = GlobalEventLog(Organizations.definition, logConfig, xas)
+
+  implicit val subject: Subject = Identity.User("Bob", Label.unsafe("realm"))
+  implicit val uuidF: UUIDF     = UUIDF.fixed(orgUuid)
+
+  test("Fail when trying to delete a non-empty organization") {
+    val arrange: IO[Any, Unit] = for {
+      _ <- truncateTables
```

Comment:
Using two different orgs for the two tests could avoid to truncate the tables and may make things easier